frontend/src/pages/PipelineVersionList.tsx

```diff
@@ -38,6 +39,11 @@ interface PipelineVersionListState {
   pipelineVersions: ApiPipelineVersion[];
 }
 
+const descriptionCustomRenderer: React.FC<CustomRendererProps<string>> = (
+  props: CustomRendererProps<string>,
+) => {
+  return <Description description={props.value || ''} forceInline={true} />;
```

Comment:
Adding Tooltip to description and version name allows user to see the full string when hovering over to them:

https://github.com/kubeflow/pipelines/blob/master/frontend/src/pages/PipelineList.tsx#L196

Reply:
Added. Had to add an extra span due to ToolTip binding not working for Description element. Adding span solved it.